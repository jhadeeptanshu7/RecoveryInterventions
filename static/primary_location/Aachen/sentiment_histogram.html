



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="7befbe7b-c650-4087-82c2-caa8d8480364"></div>
            
          
        
      
      
        <script type="application/json" id="137190">
          {"dd0959ec-96dd-4be5-bc82-cbf9e6ee33f8":{"roots":{"references":[{"attributes":{},"id":"137095","type":"LinearScale"},{"attributes":{"axis_label":"Sentiment Score","formatter":{"id":"137128","type":"CategoricalTickFormatter"},"major_label_orientation":0.7853981633974483,"plot":{"id":"137087","subtype":"Figure","type":"Plot"},"ticker":{"id":"137098","type":"CategoricalTicker"}},"id":"137097","type":"CategoricalAxis"},{"attributes":{"callback":null,"start":0},"id":"137091","type":"DataRange1d"},{"attributes":{},"id":"137093","type":"CategoricalScale"},{"attributes":{},"id":"137098","type":"CategoricalTicker"},{"attributes":{},"id":"137102","type":"BasicTicker"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"137087","subtype":"Figure","type":"Plot"},"ticker":{"id":"137098","type":"CategoricalTicker"}},"id":"137100","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"137130","type":"BasicTickFormatter"},"plot":{"id":"137087","subtype":"Figure","type":"Plot"},"ticker":{"id":"137102","type":"BasicTicker"}},"id":"137101","type":"LinearAxis"},{"attributes":{},"id":"137106","type":"PanTool"},{"attributes":{},"id":"137107","type":"WheelZoomTool"},{"attributes":{"dimension":1,"plot":{"id":"137087","subtype":"Figure","type":"Plot"},"ticker":{"id":"137102","type":"BasicTicker"}},"id":"137105","type":"Grid"},{"attributes":{"data_source":{"id":"137121","type":"ColumnDataSource"},"glyph":{"id":"137122","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"137123","type":"VBar"},"selection_glyph":null,"view":{"id":"137125","type":"CDSView"}},"id":"137124","type":"GlyphRenderer"},{"attributes":{"source":{"id":"137121","type":"ColumnDataSource"}},"id":"137125","type":"CDSView"},{"attributes":{"overlay":{"id":"137114","type":"BoxAnnotation"}},"id":"137108","type":"BoxZoomTool"},{"attributes":{},"id":"137109","type":"SaveTool"},{"attributes":{"callback":null,"data":{"fill_color":["green"],"line_color":["green"],"top":[1],"x":["0.08"]},"selected":{"id":"137131","type":"Selection"},"selection_policy":{"id":"137132","type":"UnionRenderers"}},"id":"137121","type":"ColumnDataSource"},{"attributes":{"fill_color":{"field":"fill_color"},"line_color":{"field":"line_color"},"top":{"field":"top"},"width":{"value":1},"x":{"field":"x"}},"id":"137122","type":"VBar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":1},"x":{"field":"x"}},"id":"137123","type":"VBar"},{"attributes":{"below":[{"id":"137097","type":"CategoricalAxis"}],"left":[{"id":"137101","type":"LinearAxis"}],"plot_height":250,"renderers":[{"id":"137097","type":"CategoricalAxis"},{"id":"137100","type":"Grid"},{"id":"137101","type":"LinearAxis"},{"id":"137105","type":"Grid"},{"id":"137114","type":"BoxAnnotation"},{"id":"137124","type":"GlyphRenderer"}],"title":{"id":"137086","type":"Title"},"toolbar":{"id":"137112","type":"Toolbar"},"x_range":{"id":"137089","type":"FactorRange"},"x_scale":{"id":"137093","type":"CategoricalScale"},"y_range":{"id":"137091","type":"DataRange1d"},"y_scale":{"id":"137095","type":"LinearScale"}},"id":"137087","subtype":"Figure","type":"Plot"},{"attributes":{"plot":null,"text":"Aachen - Average Sentiment Score = 0.08 and Standard Deviation = 0.0"},"id":"137086","type":"Title"},{"attributes":{"callback":null,"factors":["0.08"]},"id":"137089","type":"FactorRange"},{"attributes":{},"id":"137128","type":"CategoricalTickFormatter"},{"attributes":{},"id":"137111","type":"HelpTool"},{"attributes":{},"id":"137110","type":"ResetTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"137106","type":"PanTool"},{"id":"137107","type":"WheelZoomTool"},{"id":"137108","type":"BoxZoomTool"},{"id":"137109","type":"SaveTool"},{"id":"137110","type":"ResetTool"},{"id":"137111","type":"HelpTool"}]},"id":"137112","type":"Toolbar"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"137114","type":"BoxAnnotation"},{"attributes":{},"id":"137132","type":"UnionRenderers"},{"attributes":{},"id":"137131","type":"Selection"},{"attributes":{},"id":"137130","type":"BasicTickFormatter"}],"root_ids":["137087"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('137190').textContent;
                  var render_items = [{"docid":"dd0959ec-96dd-4be5-bc82-cbf9e6ee33f8","roots":{"137087":"7befbe7b-c650-4087-82c2-caa8d8480364"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>