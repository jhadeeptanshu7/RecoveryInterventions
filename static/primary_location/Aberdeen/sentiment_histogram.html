



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="c3e3e2ef-8e81-4011-984d-aa6f6ba46b64"></div>
            
          
        
      
      
        <script type="application/json" id="139508">
          {"917222a2-e0d5-47f9-b8f1-eff5edff0c4d":{"roots":{"references":[{"attributes":{"dimension":1,"plot":{"id":"139405","subtype":"Figure","type":"Plot"},"ticker":{"id":"139420","type":"BasicTicker"}},"id":"139423","type":"Grid"},{"attributes":{},"id":"139429","type":"HelpTool"},{"attributes":{},"id":"139428","type":"ResetTool"},{"attributes":{"below":[{"id":"139415","type":"CategoricalAxis"}],"left":[{"id":"139419","type":"LinearAxis"}],"plot_height":250,"renderers":[{"id":"139415","type":"CategoricalAxis"},{"id":"139418","type":"Grid"},{"id":"139419","type":"LinearAxis"},{"id":"139423","type":"Grid"},{"id":"139432","type":"BoxAnnotation"},{"id":"139442","type":"GlyphRenderer"}],"title":{"id":"139404","type":"Title"},"toolbar":{"id":"139430","type":"Toolbar"},"x_range":{"id":"139407","type":"FactorRange"},"x_scale":{"id":"139411","type":"CategoricalScale"},"y_range":{"id":"139409","type":"DataRange1d"},"y_scale":{"id":"139413","type":"LinearScale"}},"id":"139405","subtype":"Figure","type":"Plot"},{"attributes":{"plot":null,"text":"Aberdeen - Average Sentiment Score = 0.01 and Standard Deviation = 0.01"},"id":"139404","type":"Title"},{"attributes":{"callback":null,"factors":["0.0","0.03"]},"id":"139407","type":"FactorRange"},{"attributes":{"callback":null,"start":0},"id":"139409","type":"DataRange1d"},{"attributes":{},"id":"139420","type":"BasicTicker"},{"attributes":{},"id":"139427","type":"SaveTool"},{"attributes":{"overlay":{"id":"139432","type":"BoxAnnotation"}},"id":"139426","type":"BoxZoomTool"},{"attributes":{},"id":"139425","type":"WheelZoomTool"},{"attributes":{},"id":"139424","type":"PanTool"},{"attributes":{},"id":"139449","type":"Selection"},{"attributes":{},"id":"139448","type":"BasicTickFormatter"},{"attributes":{},"id":"139446","type":"CategoricalTickFormatter"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":0.05},"x":{"field":"x"}},"id":"139441","type":"VBar"},{"attributes":{"fill_color":{"field":"fill_color"},"line_color":{"field":"line_color"},"top":{"field":"top"},"width":{"value":0.05},"x":{"field":"x"}},"id":"139440","type":"VBar"},{"attributes":{"source":{"id":"139439","type":"ColumnDataSource"}},"id":"139443","type":"CDSView"},{"attributes":{"data_source":{"id":"139439","type":"ColumnDataSource"},"glyph":{"id":"139440","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"139441","type":"VBar"},"selection_glyph":null,"view":{"id":"139443","type":"CDSView"}},"id":"139442","type":"GlyphRenderer"},{"attributes":{},"id":"139413","type":"LinearScale"},{"attributes":{},"id":"139411","type":"CategoricalScale"},{"attributes":{},"id":"139416","type":"CategoricalTicker"},{"attributes":{"callback":null,"data":{"fill_color":["grey","green"],"line_color":["grey","green"],"top":[1,1],"x":["0.0","0.03"]},"selected":{"id":"139449","type":"Selection"},"selection_policy":{"id":"139450","type":"UnionRenderers"}},"id":"139439","type":"ColumnDataSource"},{"attributes":{"axis_label":"Sentiment Score","formatter":{"id":"139446","type":"CategoricalTickFormatter"},"major_label_orientation":0.7853981633974483,"plot":{"id":"139405","subtype":"Figure","type":"Plot"},"ticker":{"id":"139416","type":"CategoricalTicker"}},"id":"139415","type":"CategoricalAxis"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"139405","subtype":"Figure","type":"Plot"},"ticker":{"id":"139416","type":"CategoricalTicker"}},"id":"139418","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"139448","type":"BasicTickFormatter"},"plot":{"id":"139405","subtype":"Figure","type":"Plot"},"ticker":{"id":"139420","type":"BasicTicker"}},"id":"139419","type":"LinearAxis"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"139424","type":"PanTool"},{"id":"139425","type":"WheelZoomTool"},{"id":"139426","type":"BoxZoomTool"},{"id":"139427","type":"SaveTool"},{"id":"139428","type":"ResetTool"},{"id":"139429","type":"HelpTool"}]},"id":"139430","type":"Toolbar"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"139432","type":"BoxAnnotation"},{"attributes":{},"id":"139450","type":"UnionRenderers"}],"root_ids":["139405"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('139508').textContent;
                  var render_items = [{"docid":"917222a2-e0d5-47f9-b8f1-eff5edff0c4d","roots":{"139405":"c3e3e2ef-8e81-4011-984d-aa6f6ba46b64"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>