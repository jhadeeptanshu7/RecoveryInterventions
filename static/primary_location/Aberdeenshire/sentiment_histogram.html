



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="b03d4d91-5ea8-4127-8890-abd0e2ad13df"></div>
            
          
        
      
      
        <script type="application/json" id="251200">
          {"2c3f729c-8c5d-4f05-bc6b-ef7a2d82ff1b":{"roots":{"references":[{"attributes":{"below":[{"id":"251107","type":"CategoricalAxis"}],"left":[{"id":"251111","type":"LinearAxis"}],"plot_height":250,"renderers":[{"id":"251107","type":"CategoricalAxis"},{"id":"251110","type":"Grid"},{"id":"251111","type":"LinearAxis"},{"id":"251115","type":"Grid"},{"id":"251124","type":"BoxAnnotation"},{"id":"251134","type":"GlyphRenderer"}],"title":{"id":"251096","type":"Title"},"toolbar":{"id":"251122","type":"Toolbar"},"x_range":{"id":"251099","type":"FactorRange"},"x_scale":{"id":"251103","type":"CategoricalScale"},"y_range":{"id":"251101","type":"DataRange1d"},"y_scale":{"id":"251105","type":"LinearScale"}},"id":"251097","subtype":"Figure","type":"Plot"},{"attributes":{"plot":null,"text":"Aberdeenshire - Average Sentiment Score = 0.08 and Standard Deviation = 0.0"},"id":"251096","type":"Title"},{"attributes":{"callback":null,"factors":["0.08"]},"id":"251099","type":"FactorRange"},{"attributes":{"axis_label":"Sentiment Score","formatter":{"id":"251138","type":"CategoricalTickFormatter"},"major_label_orientation":0.7853981633974483,"plot":{"id":"251097","subtype":"Figure","type":"Plot"},"ticker":{"id":"251108","type":"CategoricalTicker"}},"id":"251107","type":"CategoricalAxis"},{"attributes":{},"id":"251105","type":"LinearScale"},{"attributes":{},"id":"251103","type":"CategoricalScale"},{"attributes":{"callback":null,"start":0},"id":"251101","type":"DataRange1d"},{"attributes":{},"id":"251142","type":"UnionRenderers"},{"attributes":{},"id":"251141","type":"Selection"},{"attributes":{},"id":"251140","type":"BasicTickFormatter"},{"attributes":{},"id":"251108","type":"CategoricalTicker"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"251124","type":"BoxAnnotation"},{"attributes":{},"id":"251121","type":"HelpTool"},{"attributes":{},"id":"251120","type":"ResetTool"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"251116","type":"PanTool"},{"id":"251117","type":"WheelZoomTool"},{"id":"251118","type":"BoxZoomTool"},{"id":"251119","type":"SaveTool"},{"id":"251120","type":"ResetTool"},{"id":"251121","type":"HelpTool"}]},"id":"251122","type":"Toolbar"},{"attributes":{},"id":"251117","type":"WheelZoomTool"},{"attributes":{"data_source":{"id":"251131","type":"ColumnDataSource"},"glyph":{"id":"251132","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"251133","type":"VBar"},"selection_glyph":null,"view":{"id":"251135","type":"CDSView"}},"id":"251134","type":"GlyphRenderer"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":1},"x":{"field":"x"}},"id":"251133","type":"VBar"},{"attributes":{"overlay":{"id":"251124","type":"BoxAnnotation"}},"id":"251118","type":"BoxZoomTool"},{"attributes":{},"id":"251119","type":"SaveTool"},{"attributes":{},"id":"251138","type":"CategoricalTickFormatter"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"251097","subtype":"Figure","type":"Plot"},"ticker":{"id":"251108","type":"CategoricalTicker"}},"id":"251110","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"251140","type":"BasicTickFormatter"},"plot":{"id":"251097","subtype":"Figure","type":"Plot"},"ticker":{"id":"251112","type":"BasicTicker"}},"id":"251111","type":"LinearAxis"},{"attributes":{},"id":"251112","type":"BasicTicker"},{"attributes":{"source":{"id":"251131","type":"ColumnDataSource"}},"id":"251135","type":"CDSView"},{"attributes":{"fill_color":{"field":"fill_color"},"line_color":{"field":"line_color"},"top":{"field":"top"},"width":{"value":1},"x":{"field":"x"}},"id":"251132","type":"VBar"},{"attributes":{"dimension":1,"plot":{"id":"251097","subtype":"Figure","type":"Plot"},"ticker":{"id":"251112","type":"BasicTicker"}},"id":"251115","type":"Grid"},{"attributes":{},"id":"251116","type":"PanTool"},{"attributes":{"callback":null,"data":{"fill_color":["green"],"line_color":["green"],"top":[1],"x":["0.08"]},"selected":{"id":"251141","type":"Selection"},"selection_policy":{"id":"251142","type":"UnionRenderers"}},"id":"251131","type":"ColumnDataSource"}],"root_ids":["251097"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('251200').textContent;
                  var render_items = [{"docid":"2c3f729c-8c5d-4f05-bc6b-ef7a2d82ff1b","roots":{"251097":"b03d4d91-5ea8-4127-8890-abd0e2ad13df"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>