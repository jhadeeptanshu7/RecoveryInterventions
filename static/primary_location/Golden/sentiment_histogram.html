



<!DOCTYPE html>
<html lang="en">
  
  <head>
    
      <meta charset="utf-8">
      <title>Bokeh Plot</title>
      
      
        
          
        <link rel="stylesheet" href="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.css" type="text/css" />
        
        
          
        <script type="text/javascript" src="https://cdn.pydata.org/bokeh/release/bokeh-1.0.1.min.js"></script>
        <script type="text/javascript">
            Bokeh.set_log_level("info");
        </script>
        
      
      
    
  </head>
  
  
  <body>
    
      
        
          
          
            
              <div class="bk-root" id="19e6780d-e6f2-4e3a-9688-b71018445b8c"></div>
            
          
        
      
      
        <script type="application/json" id="209162">
          {"2d5896e3-9e15-4e09-b913-ecbdcf7baeab":{"roots":{"references":[{"attributes":{"plot":null,"text":"Golden - Average Sentiment Score = 0.0 and Standard Deviation = 0.0"},"id":"209058","type":"Title"},{"attributes":{"below":[{"id":"209069","type":"CategoricalAxis"}],"left":[{"id":"209073","type":"LinearAxis"}],"plot_height":250,"renderers":[{"id":"209069","type":"CategoricalAxis"},{"id":"209072","type":"Grid"},{"id":"209073","type":"LinearAxis"},{"id":"209077","type":"Grid"},{"id":"209086","type":"BoxAnnotation"},{"id":"209096","type":"GlyphRenderer"}],"title":{"id":"209058","type":"Title"},"toolbar":{"id":"209084","type":"Toolbar"},"x_range":{"id":"209061","type":"FactorRange"},"x_scale":{"id":"209065","type":"CategoricalScale"},"y_range":{"id":"209063","type":"DataRange1d"},"y_scale":{"id":"209067","type":"LinearScale"}},"id":"209059","subtype":"Figure","type":"Plot"},{"attributes":{"bottom_units":"screen","fill_alpha":{"value":0.5},"fill_color":{"value":"lightgrey"},"left_units":"screen","level":"overlay","line_alpha":{"value":1.0},"line_color":{"value":"black"},"line_dash":[4,4],"line_width":{"value":2},"plot":null,"render_mode":"css","right_units":"screen","top_units":"screen"},"id":"209086","type":"BoxAnnotation"},{"attributes":{"active_drag":"auto","active_inspect":"auto","active_multi":null,"active_scroll":"auto","active_tap":"auto","tools":[{"id":"209078","type":"PanTool"},{"id":"209079","type":"WheelZoomTool"},{"id":"209080","type":"BoxZoomTool"},{"id":"209081","type":"SaveTool"},{"id":"209082","type":"ResetTool"},{"id":"209083","type":"HelpTool"}]},"id":"209084","type":"Toolbar"},{"attributes":{},"id":"209083","type":"HelpTool"},{"attributes":{},"id":"209082","type":"ResetTool"},{"attributes":{},"id":"209081","type":"SaveTool"},{"attributes":{"overlay":{"id":"209086","type":"BoxAnnotation"}},"id":"209080","type":"BoxZoomTool"},{"attributes":{"axis_label":"Sentiment Score","formatter":{"id":"209100","type":"CategoricalTickFormatter"},"major_label_orientation":0.7853981633974483,"plot":{"id":"209059","subtype":"Figure","type":"Plot"},"ticker":{"id":"209070","type":"CategoricalTicker"}},"id":"209069","type":"CategoricalAxis"},{"attributes":{},"id":"209104","type":"UnionRenderers"},{"attributes":{},"id":"209102","type":"BasicTickFormatter"},{"attributes":{},"id":"209103","type":"Selection"},{"attributes":{},"id":"209100","type":"CategoricalTickFormatter"},{"attributes":{"callback":null,"factors":["0.0"]},"id":"209061","type":"FactorRange"},{"attributes":{"callback":null,"start":0},"id":"209063","type":"DataRange1d"},{"attributes":{},"id":"209065","type":"CategoricalScale"},{"attributes":{},"id":"209067","type":"LinearScale"},{"attributes":{"callback":null,"data":{"fill_color":["grey"],"line_color":["grey"],"top":[1],"x":["0.0"]},"selected":{"id":"209103","type":"Selection"},"selection_policy":{"id":"209104","type":"UnionRenderers"}},"id":"209093","type":"ColumnDataSource"},{"attributes":{"fill_color":{"field":"fill_color"},"line_color":{"field":"line_color"},"top":{"field":"top"},"width":{"value":1},"x":{"field":"x"}},"id":"209094","type":"VBar"},{"attributes":{"fill_alpha":{"value":0.1},"fill_color":{"value":"#1f77b4"},"line_alpha":{"value":0.1},"line_color":{"value":"#1f77b4"},"top":{"field":"top"},"width":{"value":1},"x":{"field":"x"}},"id":"209095","type":"VBar"},{"attributes":{"data_source":{"id":"209093","type":"ColumnDataSource"},"glyph":{"id":"209094","type":"VBar"},"hover_glyph":null,"muted_glyph":null,"nonselection_glyph":{"id":"209095","type":"VBar"},"selection_glyph":null,"view":{"id":"209097","type":"CDSView"}},"id":"209096","type":"GlyphRenderer"},{"attributes":{"source":{"id":"209093","type":"ColumnDataSource"}},"id":"209097","type":"CDSView"},{"attributes":{},"id":"209078","type":"PanTool"},{"attributes":{},"id":"209079","type":"WheelZoomTool"},{"attributes":{"grid_line_color":{"value":null},"plot":{"id":"209059","subtype":"Figure","type":"Plot"},"ticker":{"id":"209070","type":"CategoricalTicker"}},"id":"209072","type":"Grid"},{"attributes":{"axis_label":"Number of Drug Users","formatter":{"id":"209102","type":"BasicTickFormatter"},"plot":{"id":"209059","subtype":"Figure","type":"Plot"},"ticker":{"id":"209074","type":"BasicTicker"}},"id":"209073","type":"LinearAxis"},{"attributes":{},"id":"209070","type":"CategoricalTicker"},{"attributes":{"dimension":1,"plot":{"id":"209059","subtype":"Figure","type":"Plot"},"ticker":{"id":"209074","type":"BasicTicker"}},"id":"209077","type":"Grid"},{"attributes":{},"id":"209074","type":"BasicTicker"}],"root_ids":["209059"]},"title":"Bokeh Application","version":"1.0.1"}}
        </script>
        <script type="text/javascript">
          (function() {
            var fn = function() {
              Bokeh.safely(function() {
                (function(root) {
                  function embed_document(root) {
                    
                  var docs_json = document.getElementById('209162').textContent;
                  var render_items = [{"docid":"2d5896e3-9e15-4e09-b913-ecbdcf7baeab","roots":{"209059":"19e6780d-e6f2-4e3a-9688-b71018445b8c"}}];
                  root.Bokeh.embed.embed_items(docs_json, render_items);
                
                  }
                  if (root.Bokeh !== undefined) {
                    embed_document(root);
                  } else {
                    var attempts = 0;
                    var timer = setInterval(function(root) {
                      if (root.Bokeh !== undefined) {
                        embed_document(root);
                        clearInterval(timer);
                      }
                      attempts++;
                      if (attempts > 100) {
                        console.log("Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing");
                        clearInterval(timer);
                      }
                    }, 10, root)
                  }
                })(window);
              });
            };
            if (document.readyState != "loading") fn();
            else document.addEventListener("DOMContentLoaded", fn);
          })();
        </script>
    
  </body>
  
</html>